native/html5ever_nif/Cargo.toml

```diff
@@ -7,10 +7,10 @@ edition = "2018"
 [lib]
 name = "html5ever_nif"
 path = "src/lib.rs"
-crate-type = ["dylib"]
+crate-type = ["cdylib"]
```

Comment:
This is the crate type recommended for shared libs that are not going to be used in Rust programs. See https://doc.rust-lang.org/reference/linkage.html

Reply:
Nice catch!